terraso_backend/apps/soilproj/service.py

```diff
@@ -0,0 +1,12 @@
+
+from typing import Optional
+
+from . import models
+
+
+class SoilService():
+
+    def get_site_by_name(self, name: str) -> Optional[models.Site]:
```

Comment:
Question: Can you explain what is ` -> Optional[models.Site]` doing here? I am not familiar with this notation